mkdocs.yml

```diff
@@ -490,6 +490,11 @@ plugins:
               "release-notes/cloud/scheduled-maintenance-saturday,-30th-mar@-05:00-gmt-(01:00-edt).md": "index.md"
               "release-notes/cloud/scheduled-maintenance-saturday,-17th-nov@8:00-gmt-(0:00-pst).md": "index.md"
               "faq/repositories/how-do-i-migrate-a-legacy-repository-to-a-synced-organization.md": "release-notes/cloud/cloud-2021-11-02-legacy-organizations.md"
+              "organizations/manual-organizations/creating-and-renaming-an-organization.md": "release-notes/cloud/cloud-2021-11-02-legacy-organizations.md"
+              "organizations/manual-organizations/creating-and-managing-teams.md": "release-notes/cloud/cloud-2021-11-02-legacy-organizations.md"
+              "organizations/manual-organizations/managing-team-repositories.md": "release-notes/cloud/cloud-2021-11-02-legacy-organizations.md"
+              "organizations/manual-organizations/administrative-permissions.md": "release-notes/cloud/cloud-2021-11-02-legacy-organizations.md"
+              "organizations/manual-organizations/share-your-repository-with-a-non-codacy-user.md": "release-notes/cloud/cloud-2021-11-02-legacy-organizations.md"
```

Comment:
I really like this idea of redirecting the removed pages to the announcement in the release notes! :raised_hands: